x/ibc/20-transfer/keeper/keeper.go

```diff
@@ -66,3 +62,7 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) GetTransferAccount(ctx sdk.Context) supplyexported.ModuleAccountI {
 	return k.supplyKeeper.GetModuleAccount(ctx, types.GetModuleAccountName())
 }
+
+func (k Keeper) PacketExecuted(ctx sdk.Context, packet channelexported.PacketI, acknowledgement channelexported.PacketDataI) error {
```

Comment:
is this wrapper function really needed? where is it being used?

Reply:
I updated the ICS 20 code to use it. It should be used.

Reply:
It's used in handler.go (line 66)